xikolo-common/UserDefaultsKeys.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UserDefaultsKeys.swift
+//  xikolo-ios
+//
+//  Created by Max Bothe on 26.07.17.
+//  Copyright © 2017 HPI. All rights reserved.
+//
+
+import Foundation
+
+struct UserDefaultsKeys {
+
+    // negative form to have content preload activated by default
+    // since UserDefault will return `false` if no value for the key is present
+    static let noContentPreloadKey = "de.xikolo.ios.content.noPreload"
```

Comment:
The only other idea would be to check wether it is the first launch of the app and set this setting in the first launch manually.